history/characters/11000_dark_iron.txt

```diff
@@ -1150,14 +1150,23 @@
 	369.5.7={
 		effect={ set_variable = { name = wc_elemental_fire_magic_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_3_magic_trait_value } }
 	}
-	608.1.1={
-		add_spouse=17029
+	608.12.22 = {
+		add_spouse = 17029 # Moira
 		effect = {
 			set_relation_soulmate = character:17029
 		}
 	}
 	640.2.6={ death=yes }
 }
+dagran2={
+	name = Dagran # Dagran Thaurissan II
+	dynasty = 4010
+	culture = dark_iron
+	religion = fire_cult
+	father = 12002 # Dagran Thaurissan
+	mother = 17029 # Moira Bronzebeard
+	609.1.8={ birth=yes trait=creature_dwarf effect = { make_important_lore_character_effect = yes } }
+}
```

Comment:
That would mean that Moira gave birth to Dagran the II in less than 2 weeks after marrying Dagran

Reply:
I would say you can change all things through commits, your branch if you ok. Because not my work x)